ui/templates/main.html

```diff
@@ -9,6 +12,8 @@ <h1>Φορτίο (fortio) v{{.Version}}{{if not .DoLoad}} control UI{{end}}</h1>
 <p>Up for {{.UpTime}} (since {{.StartTime}}).
 {{if .DoLoad}}
 <p>{{.Labels}} {{.TargetURL}}
+<div id="progressBarDiv"></div>  
```

Comment:
maybe this could be <progress> instead ?

like 
https://github.com/chartjs/Chart.js/blob/master/samples/advanced/progress-bar.html
http://www.chartjs.org/samples/latest/advanced/progress-bar.html

Reply:
that's a good idea, I can experiment with that if you like!